pep-0676.rst

```diff
@@ -121,69 +128,84 @@ The following redirect rules must be created from the `python.org`_ domain:
         return 308 https://peps.python.org/;
     }
 
-Redirects must be implemented to preserve `URL fragments`_ for backward
-compatability purposes.
+Redirects MUST be implemented to preserve `URL fragments`_ for backward
+compatibility purposes.
+
 
 Backwards Compatibility
 =======================
 
-Due to server-side redirects to new canonical URLs, there are no backwards
-compatability concerns. Links in previously published materials referring to
-any old URL scheme will be guaranteed to work.
+Due to server-side redirects to new canonical URLs, links in previously
+published materials referring to the old URL scheme will be guaranteed to work.
+Furthermore, aside from the resolved bugs and new implemented features, all
+existing PEPs will still render as they did before with the new backend.
+Therefore, this poses no major backwards compatibility concerns.
+
 
 Security Implications
 =====================
 
-No security implications, and the main `python.org`_ website will no longer
-take raw HTML uploads, closing a potential threat vector.
+The main `python.org`_ website will no longer need to process raw HTML
+uploads, closing a potential threat vector, and the PEP building and
+deployment process will use modern, well maintained code and secure
+automated platforms, reducing the likely potential attack surface. Therefore,
+there is no forseen negative security impact.
+
 
 How to Teach This
 =================
 
 The new canonical URLs will be publicised in the documentation. However, this
 is mainly a backend infrastructure change, and there should be minimal
-end-user impact.
+end-user impact. PEP 1 and PEP 12 will be updated as needed to include and
+document any relevant new roles, directive, syntax and other constructs enabled
+by this change.
+
 
 Reference Implementation
 ========================
 
 The proposed implementation has been merged into the `python/peps`_ repository
-in a series of pull requests [8]_. This automatically renders all PEPs on every
-commit.
+in a series of pull requests [8]_. It currently automatically renders all
```

Comment:
I can see the issue with `currently`, but the issue here is it is not clear if the implementation is merely _capable_ of automatically rendering all PEPs, or is actually already actively in use doing (which makes a stronger argument in this PEP's favor). What about "already" instead?

```suggestion
in a series of pull requests [8]_. This already automatically renders all
```

Or what about this, which would be cleaner but I'd need to apply it manually since its over the line length limit:

```suggestion
in a series of pull requests [8]_. It is already used to automatically render all
```